data/serializer/src/main/java/tech/pegasys/teku/api/request/v1/validator/BeaconCommitteeSubscriptionRequest.java

```diff
@@ -37,8 +37,8 @@ public class BeaconCommitteeSubscriptionRequest {
 
   @JsonCreator
   public BeaconCommitteeSubscriptionRequest(
-      @JsonProperty("validator_index") final int validator_index,
-      @JsonProperty("committee_index") final int committee_index,
+      @JsonProperty("validator_index") final String validator_index,
+      @JsonProperty("committee_index") final String committee_index,
```

Comment:
wonder if we want to CheckNotNull on these two now that its nullable... maybe a test to show we handle things ok if these fields aren't set - should be a 400 not a 500 error...

Reply:
may translate to an illegalArgumentException depending on where you are maybe...

Reply:
thank you, added `checkNotNull` preconditon to both fields